sdk-extensions/incubator/build.gradle.kts

```diff
@@ -26,6 +26,7 @@ dependencies {
 
   // io.opentelemetry.sdk.extension.incubator.fileconfig
   implementation("com.fasterxml.jackson.core:jackson-databind")
+  api("com.fasterxml.jackson.core:jackson-annotations")
```

Comment:
Without this, https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1440 is full of warnings like:
```
/Users/jberg/.m2/repository/io/opentelemetry/opentelemetry-sdk-extension-incubator/1.42.0-alpha-SNAPSHOT/opentelemetry-sdk-extension-incubator-1.42.0-alpha-SNAPSHOT.jar(/io/opentelemetry/sdk/extension/incubator/fileconfig/internal/model/OpenTelemetryConfiguration.class): warning: Cannot find annotation method 'value()' in type 'JsonProperty'
```